aries-site/src/layouts/content/Subsection.js

```diff
@@ -22,22 +29,50 @@ export const Subsection = ({ children, name, ...rest }) => {
       onFocus={() => setOver(true)}
       onBlur={() => setOver(false)}
     >
-      <Box direction="row" justify="between">
-        <Subheading {...rest}>{name}</Subheading>
-        <Anchor
-          a11yTitle={`Jump to section titled ${name}`}
-          href={`#${id}`}
-          icon={<LinkIcon color={over ? 'text-xweak' : 'transparent'} />}
-        />
+      {/* This condition for gap is needed because the link icon has padding
+       * to maintain a big enough touch area for accessibility. This creates
+       * a larger than desired gap between h3's and its first child. This
+       * removes that extra space.
+       */}
+      <Box gap={GAP_SIZE[level]}>
+        <Box direction="row" justify="between" align="center">
```

Comment:
I think this is a good candidate for the Header component. Header has `direction="row" justify="between" align="center"` embedded in it.

Reply:
It does sound like a good usage, let me try that out!